app/mailers/candidate_mailer.rb

```diff
@@ -457,6 +457,8 @@ def new_cycle_has_started(application_form)
   end
 
   def fraud_match_email(application_form)
+    @application_form = application_form
+    @application_submitted = !@application_form.submitted_at.nil?
```

Comment:
I think we need to check this sooner? ie if I have two forms in progress, one with email A and one with email B.

The SendFraudMatchEmail service runs and emails both A and B.

A gets an email saying 'one of your applications has been submitted'
B gets an email saying 'your access to the account you set up most recently has been removed'

At the point where we loop over the candidates, perhaps we should determine whether either has been submitted, and then pass that information into to this method?